outrank/algorithms/feature_ranking/ranking_mi_numba.py

```diff
@@ -117,6 +115,37 @@ def compute_entropies(
         return core_joint_entropy
 
 
+@njit(
+    'Tuple((int32[:], int32[:]))(int32[:], int32[:], float32, int32[:])',
+)
+def stratified_subsampling(Y, X, approximation_factor, f_values_X):
```

Comment:
No, because this is the output of a function that guarantees uniqueness. If it turns out anyone ever uses this function in isolation, we can add the check

Reply:
good idea, done